src/tests/ftest/rebuild/rebuild_container_rf.yaml

```diff
@@ -22,7 +22,11 @@ pool:
   pool_query_timeout: 30
 container:
   control_method: daos
-  properties: rf:2
+  cont_prop: !mux
```

Comment:
I think the definition of negative/positive test in lines 37-46 is confusing in this file. 
Technically for rf0 in this case, they should be negative and negative tests because you should be expecting UNCLEAN in both cases.
Maybe consider something like this:
add rf_1 to cont_prop
in cont_rf, consider
   kill_1:
      rank:
         -3
   kill_2
      rank:
         -3
        -4
   kill_3
      rank:
         -3
         -4
         -5

Reply:
ok.
Please see the new commit.